build/slicer/index.md

```diff
@@ -15,15 +15,15 @@ While using different slicers for different kinds of prints can be beneficial av
 
 ## Slicer Selection
 
-There are two slicers that come with built-in profiles for most Voron printers:
+There are three slicers that come with built-in profiles for most Voron printers:
 
 * Cura
```

Comment:
I may be biased, but do we want cura at the top of this list, given how few of us supporters actually use it

Reply:
also fixed